spec/controllers/articles_controller_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
+  describe 'GET #index' do
     describe 'show the list to articles saved' do
     end
   end
 
-  describe '#show' do
+  describe 'GET #show' do
     it 'show the articles' do
     end
   end
 
-  describe '#new'do
+  describe 'GET #new'do
     it 'create new post' do
     end
   end
 
-    describe ' #create' do
+    describe 'GET #create' do
         context 'when invalid an article' do
```

Comment:
If you are trying to model various context, almost every time it has opposite context, for example:

`context 'when article is valid' do` and also `context 'when article is invalid' do`.